addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/ParameterTamperScanRule.java

```diff
@@ -62,7 +62,7 @@
     private static Pattern patternErrorJava2 =
             Pattern.compile("invoke.+exception|exception.+invoke", PATTERN_PARAM);
     private static Pattern patternErrorVBScript =
-            Pattern.compile("Microsoft(\\s+|&nbsp)*VBScript(\\s+|&nbsp)+error", PATTERN_PARAM);
+            Pattern.compile("Microsoft(?:\\s|&nbsp)+VBScript(?:\\s|&nbsp)+error", PATTERN_PARAM);
```

Comment:
This changes the meaning of the regular expression, before no space/entity was required between `Microsoft` and `VBScript`, now it is (the change might be ok, but should be noted and tested).

Reply:
There definitely would have been a space in any actual response. I'll update the CHANGELOG.